app/src/main/java/com/example/dashcarr/presentation/authentication/login/LoginFragment.kt

```diff
@@ -28,7 +28,8 @@ import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class LoginFragment : BaseFragment<FragmentLoginBinding>(
-    FragmentLoginBinding::inflate
+    FragmentLoginBinding::inflate,
+    showBottomNavBar = false
```

Comment:
I believe that Simon wanted you to remove this so that you didn't have to log in to test the app.
Maybe check with him first, before you change it permanently

Reply:
done